modules/ROOT/pages/software-extensibility.adoc

```diff
@@ -16,26 +16,23 @@ These extension points are guaranteed to be stable across versions of Bonita 7:
 
 === Connectors
 
-In Community, Teamwork, Efficiency, Performance and Enterprise editions
-
 Bonita comes with more than 80 xref:connectivity-overview.adoc[standard connectors] to the major information system components: major databases (Oracle, Microsoft, Postgres, etc.), SOAP Webservice, Salesforce, Email, etc.
-A new connector enables you to provide new connectivity capabilities for processes.
```

Comment:
We should also talk about the connectors that do not connect anywhere but are tools that need to be created as connectors: Word templating, Word to pdf transformation, script...